docs/reference/cms-dev-server.md

```diff
@@ -21,6 +21,49 @@ You may also start the dev server with the `--ssl` option, which enables:
 - https://cmssite.com.hslocal.net:3000/page
 - https://cmssite.com.localhost:3000/page
 
+## Routes
+
+When the CMS Dev Server for CMS React starts it will look at the directory structure and look for `partials` and `modules` within `components`. It will then dynamically create routes based on what it finds there.
+
+### Modules
+
+The CMS Dev Server offers two different rotes for your modules; `/module/[module_name]` and `/preview/[module_name]`.
+
+#### `/module/[module_name]`
+
+The `/module/[module_name]` route is rendered entirely locally without talking to the HubSpot backend.
+
+You can work "offline" and "unauthenticated" at this route - with caveats for GraphQL.
+
+Field values that are used are derived entirely from the Field default values and from parameter level overrides. Param level overrides can be passed via `fields` param which expects stringified JSON that would match the fields definition.
```

Comment:
Maybe "stringified JSON of fieldValues that matches what is the passed `fieldValues` prop (matching the fields definition structure)"?